src/chrome/content/rules/MSN.xml

```diff
@@ -202,34 +196,20 @@
 	<target host="byfiles.storage.msn.com" />
 
 	<target host="udc.msn.com" />
-	<target host="catalog.video.msn.com" />
 	<target host="*.ssl.catalog.video.msn.com" />
-	<!--target host="www.msn.com" /-->
 
 	<!--	Complications:
 				-->
 	<target host="h.msn.com" />
 	<target host="imgs1-video.msn.com" />
-	<target host="c.jp.msn.com" />
```

Comment:
This host is still up. If you cannot find a valid domain, please add it to the comments on the top.